external/elasticsearch/ES_IndexInit.sh

```diff
@@ -107,6 +107,51 @@ curl -XPOST localhost:9200/metrics -d '
 }
 '
 
+# uncomment below if using per day/month indices pattern
+#curl -XPOST localhost:9200/_template/storm-metrics-template -d '
+#{
+#  "template": "metrics-*",
```

Comment:
can't we modify the template pattern so that it can be used on metrics*? In that case, we could uncomment the whole lot and remove the previous section